src/app/services/baw-api/harvest/harvest.service.spec.ts

```diff
@@ -33,4 +37,26 @@ describe("HarvestsService", () => {
     harvestId, // harvest
     5 //project
   );
+
+  describe("isAbortable", () => {
+    const harvestTransitionalStates: HarvestStatus[] = [
+      "metadataReview",
+      "uploading"
+    ];
```

Comment:
you don't test the other stages?

try making two sets - one with allowed states and one with disallowed states